tests/default_vars.sh

```diff
@@ -1243,6 +1322,7 @@ export FHMAX=24
 export THRD=1
 export FHROT=0
 export WARM_START=.false.
+export WRITE_NSFLIP=.false.
```

Comment:
Please set the WRITE_NSFLIP default to .true.

Reply:
done.